requirements.txt

```diff
@@ -1,4 +1,4 @@
-ansible-lint==3.5.1
+ansible-lint>=4.0.1
```

Comment:
No, it'd influence other things in env. Maybe we could add `,<5`.

Reply:
```suggestion
ansible-lint>=4.0.1,5
```

Reply:
OK, cool.

> it'd influence other things in env.

Curious as to what.

Reply:
It's just common knowledge about Python envs. If a user installs some other software having some of its dependencies (even indirect) overlapping with Molecule which have incompatible version constraints it'd be a painful experience for them.

Reply:
That's why, for dists shared via PyPI it's common to have open requirements constraints.
OTOH pinned constraints are good, they just have a different use-case. With them you're supposed to "lock" the environment where you install software so that it's reprocucible.

Reply:
Molecule uses `pbr` currently which wierdly mixes requirements files concepts reusing them for dist deps. So it's a bit confusing when you try to identify what goes where.
I hope I'll get rid of `pbr` at some point in favor of more classical approach.

Reply:
Hey @themr0c, could you please "Apply" that suggestion above? (You can do it right from the UI)

Reply:
@webknjaz done. And if I follow you, that's the complete requirements.txt that should get adapted soon to have more open requirements constraints.